src/main/com/leandro/it/FibonacciFunction.java

```diff
@@ -0,0 +1,25 @@
+package com.leandro.it;
+
+import exceptions.InvalidNumberException;
+
+public class FibonacciFunction {
+
+    int n;
+
+    public FibonacciFunction() {
+
+    }
+
+    public int fibonacciN(int n) {
+
+        if (n < 0) {
```

Comment:
n<0 could be replaced by isValidArgument or isValidInput or checkIsParameterIsValid or something better than n<0...